### PR TITLE
1408 create and use default rules for all creates and updates

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
Case Processor is now using FieldWorkHelper class, so lots of tweaks to the ATs

# What has changed
Tweaking sample files and expected printfiles in 

# How to test?
With caseprocessor from this ticket

# Links
https://trello.com/c/TEbkKVfG/1558-change-logic-to-send-create-messages-to-field-for-household-new-addresses-5

# Screenshots (if appropriate):

### CODE REVIEW

File: resources/sample_files/sample_1_english_HH_unit.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
 UPRN,ESTAB_UPRN,ADDRESS_TYPE,ESTAB_TYPE,ADDRESS_LEVEL,ABP_CODE,ORGANISATION_NAME,ADDRESS_LINE1,ADDRESS_LINE2,ADDRESS_LINE3,TOWN_NAME,POSTCODE,LATITUDE,LONGITUDE,OA,LSOA,MSOA,LAD,REGION,HTC_WILLINGNESS,HTC_DIGITAL,TREATMENT_CODE,FIELDCOORDINATOR_ID,FIELDOFFICER_ID,CE_EXPECTED_CAPACITY,CE_SECURE,PRINT_BATCH
-123456789123,92128659213,HH,Household,U,RD06,,Flat 56 Francombe House,Commercial Road,Tree area,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,HH_LFNR1E,,,5,1,18
```
Comment (liamtoozer) [inline resources/sample_files/sample_1_english_HH_unit.csv:2]: If we didn't want to jiggery-pokery this file, we could use this one instead maybe: `sample_1_english_HH_unit_with_fieldcoordinator.csv`

File: resources/sample_files/sample_1_english_SPG_unit.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
 UPRN,ESTAB_UPRN,ADDRESS_TYPE,ESTAB_TYPE,ADDRESS_LEVEL,ABP_CODE,ORGANISATION_NAME,ADDRESS_LINE1,ADDRESS_LINE2,ADDRESS_LINE3,TOWN_NAME,POSTCODE,LATITUDE,LONGITUDE,OA,LSOA,MSOA,LAD,REGION,HTC_WILLINGNESS,HTC_DIGITAL,TREATMENT_CODE,FIELDCOORDINATOR_ID,FIELDOFFICER_ID,CE_EXPECTED_CAPACITY,CE_SECURE,PRINT_BATCH
-123456789123,66484463076,SPG,Household,U,RD06,,Flat 56 Francombe House,Commercial Road,Tree area,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,SPG_LPHUE,,,5,0,52
```
Comment (liamtoozer) [inline resources/sample_files/sample_1_english_SPG_unit.csv:2]: Could use this instead maybe? `sample_1_english_SPG_unit_with_fieldcoordinator.csv`

**Review by liamtoozer — APPROVED** [review]
Paired with Luke, looks good 👍

**Review by NickGrantX — APPROVED** [review]
Looks good

**Pull request merged**